flocker/control/_protocol.py

```diff
@@ -240,6 +243,19 @@ class ClusterStatusCommand(Command):
     response = []
 
 
+class SetNodeEraCommand(Command):
+    """
+    Set the era for a node.
+
+    This should clear any previous NodeState that has a different
+    era. Updates to the node should only be sent after this command, to
+    ensure it doesn't get stale pre-reboot information (i.e. NodeState
+    with wrong era).
```

Comment:
Does this mean that if we send updates to the node before setting the era, we'll silently corrupt the data? Is there any way we can make this more fool-proof?